sdk/resourcemanager/api-specs.json

```diff
@@ -100,7 +100,7 @@
     "dir": "azure-resourcemanager-containerservice",
     "source": "specification/containerservice/resource-manager/readme.md",
     "package": "com.azure.resourcemanager.containerservice",
-    "args": "--payload-flattening-threshold=1 --tag=package-2020-11 --title=ContainerServiceManagementClient --description=\"Container Service Client\""
+    "args": "--payload-flattening-threshold=1 --tag=package-2020-11 --title=ContainerServiceManagementClient --description=\"Container Service Client\" --remove-model=BaseManagedCluster,ManagedClusterPropertiesIdentityProfile,ManagedClusterPropertiesIdentityProfileValue --rename-model=ManagedClusterIdentityUserAssignedIdentitiesValue:ManagedClusterIdentityUserAssignedIdentities"
```

Comment:
The rename part `ManagedClusterIdentityUserAssignedIdentitiesValue:ManagedClusterIdentityUserAssignedIdentities` is in case python change 2020-11 in future.